common/src/main/cpp/include/extensions/openxr_fb_hand_tracking_capsules_extension_wrapper.h

```diff
@@ -70,6 +69,12 @@ class OpenXRFbHandTrackingCapsulesExtensionWrapper : public OpenXRExtensionWrapp
 	static void _bind_methods();
 
 private:
+	enum Hand {
+		HAND_LEFT,
+		HAND_RIGHT,
+		HAND_MAX,
+	};
```

Comment:
You're reusing `OpenXRFbHandTrackingMesh::Hand` below in `openxr_fb_hand_tracking_mesh_extension_wrapper.h`; is there a reason for not doing the same here?

Also should the `OpenXRFbHandTrackingMesh::Hand` enum be moved to `util.h`?